internal/interpreters/r_interpreter_functional_test.go

```diff
@@ -37,11 +36,12 @@ func TestRInterpreterFunctionalSuite(t *testing.T) {
 
 func (s *RInterpreterFunctionalSuite) SetupTest() {
 	// Set up the test directory
-	parentDir, err := os.MkdirTemp("", "r-interpreter-test-*")
+	// We can't use an in memory filesystem here because we will end up chdiring to it
+	fs := afero.NewOsFs()
```

Comment:
@tdstein So this actually was causing errors using the in memory file system since we can't ch into it. This might actually mean these changes are silly anyway?

Reply:
Ah. This might not work since the source code isn't also using afero.

Reply:
Yeah, I'm going to close this then, we can come back and cleanup the operative code later.